core/src/main/java/org/elasticsearch/index/shard/IndexShard.java

```diff
@@ -408,8 +418,17 @@ public IndexShard relocated(String reason) throws IndexShardNotStartedException
             if (state != IndexShardState.STARTED) {
```

Comment:
For future readers - this comment refers to another iteration - github got confused. things look good to me now.